docs/hello_nextflow/00_orientation.md

```diff
@@ -7,7 +7,8 @@ If you have not yet done so, please follow [this link](../../envsetup/) before g
 
 ## Materials provided
 
-Throughout this training course, we'll be working in the `hello-nextflow/` directory, which loads by default when you open the training workspace.
+Throughout this training course, we'll be working in the `hello-nextflow/` directory, which you can load by typing `code hello-nextflow/` in the terminal when you open the training workspace.
+Otherwise you can change directory and go into this folder with `cd hello-nextflow/`.
```

Comment:
Good catch!-- But this should be the instruction (remove the `code` option). For beginner training, just give people one way to do a thing whenever possible, ideally the most familiar way, even if there are multiple ways.

Reply:
I took the liberty of making some additional tweaks. Will merge once tests pass.